src/analysis/classic/hunter/shared/lowRankSpells.ts

```diff
@@ -0,0 +1,14 @@
+import SPELLS from 'common/SPELLS/classic/druid'; // UPDATE THE CLASS
```

Comment:
Update the class, remove the comment

Reply:
It was because I had two versions but only pointing to the one outside of shared. Replaced this file with the correct one and pointed to shared version.
Done